Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Simple usage with a mounted data directory:
-# > docker build --platform="linux/amd64" -t simapp .
+# > docker build --platform="linux/amd64" -t simapp . --build-arg ARCH=x86_64
```

Comment:
`--build-arg ARCH=x86_64` option is requirement? 
If then, how about changing hardcode without input option?

Reply:
I would prefer to enter x86_64 as an option rather than hardcode it in the Dockerfile, as it is confusing to change the `ARCH` in the Dockerfile for each architecture.

If you want to run the docker build without the option, how about adding a task to the Makefile?

```
ARCH ?= x86_64 // default value
build-docker: go.sum $(BUILDDIR)/
	docker build -t simapp . --build-arg ARCH=$(ARCH)
```

Reply:
Your idea is good. 
And how about adding `latest` as the `simapp` tag like below. 
```
          docker build -t simapp:latest . --build-arg ARCH=$(ARCH)
```